source/_webhooks.html.md.erb

```diff
@@ -459,6 +484,7 @@ For more information, see [Intro to Webhooks](#intro-to-webhooks).
 **sharedSecret** | string | Shared secret for this webhook, randomly generated by Smartsheet. Read-only. See [Authenticating Callbacks](#authenticating-callbacks-optional) for details about how this value can be used.
 **stats** | [WebhookStats](#webhookstats-object) | Useful statistics for this webhook.  Only returned once this webhook has performed a callback attempt.  Read-only.
 **status** | string | Webhook status. Read-only. See [Webhook Status](#webhook-status) for list of possible values.
+**subscope** | [columnIds](#columnIds-object)[] | Array of columnIds if you want to limit the subscription to a subscope. Specified when a webhook is created and cannot be changed.
```

Comment:
As per my comment above, rather than linking to `#columnIds-object`, is there some way to say that this is just an array of numbers?

Reply:
Yes. I was struggling with this yesterday. I decided to throw it over to reviewers and see what happened. LOL. I will simplify.